test/celeritas/ext/GeantImporter.test.cc

```diff
@@ -1293,6 +1303,91 @@ TEST_F(OneSteelSphereGG, physics)
+    auto const lar_iter = imported.optical.find(lar_idx);
+    EXPECT_FALSE(lar_iter == imported.optical.end());
+    auto const& optical = lar_iter->second;
+
+    real_type const tol = this->comparison_tolerance();
+
+    // Check scintillation optical properties
+    auto const& scint = optical.scintillation;
+    EXPECT_EQ(1, scint.resolution_scale);
+    EXPECT_EQ(5, scint.yield);
+    EXPECT_EQ(3, scint.components.size());
+    std::vector<double> components;
+    for (auto const& comp : scint.components)
+    {
+        components.push_back(comp.yield);
+        components.push_back(comp.lambda_mean);
+        components.push_back(comp.lambda_sigma);
+        components.push_back(comp.rise_time);
+        components.push_back(comp.fall_time);
+    }
+    // clang-format off
+    static double const expected_components[]
+        = {3, 1.28e-05, 1e-06, 1e-08, 6e-09,
+           1, 1.28e-05, 1e-06, 1e-08, 1.5e-06,
+           1, 2e-05,    2e-06, 1e-08, 3e-06};
+    // clang-format on
+    EXPECT_VEC_NEAR(expected_components, components, tol);
+
+    // Check common optical properties
+    auto const& properties = optical.properties;
+    EXPECT_TRUE(properties);
+    static double const expected_ri_x[] = {1.55e-06,
+                                           1.79505e-06,
+                                           2.10499e-06,
+                                           2.27077e-06,
+                                           2.55111e-06,
+                                           2.84498e-06,
+                                           3.06361e-06,
+                                           4.13281e-06,
+                                           6.2e-06,
+                                           6.526e-06,
+                                           6.889e-06,
+                                           7.294e-06,
+                                           7.75e-06,
+                                           8.267e-06,
+                                           8.857e-06,
+                                           9.538e-06,
+                                           1.033e-05,
+                                           1.55e-05};
+    static double const expected_ri_y[] = {1.4781,
+                                           1.48,
+                                           1.4842,
+                                           1.4861,
+                                           1.4915,
+                                           1.4955,
+                                           1.4988,
+                                           1.5264,
+                                           1.6185,
+                                           1.6176,
+                                           1.527,
+                                           1.5545,
+                                           1.793,
+                                           1.7826,
+                                           1.6642,
```

Comment:
In the earlier, there was a check that the index of refraction is monotonically increasing.  Is this intentionally try to make the check failure?

Reply:
Good catch! No, this was not intentional :)